staging/src/k8s.io/apimachinery/pkg/util/intstr/intstr.go

```diff
@@ -124,8 +124,7 @@ func (_ IntOrString) OpenAPIDefinition() openapi.OpenAPIDefinition {
 	return openapi.OpenAPIDefinition{
 		Schema: spec.Schema{
 			SchemaProps: spec.SchemaProps{
-				Type:   []string{"string"},
-				Format: "int-or-string",
+				Type: []string{"integer", "string"},
```

Comment:
I've tried to look at the OpenAPI spec, JSON spec, etc. to see how this is defined and also looked at tools like `swagger-codegen` to see if more than one type is allowed or more supported. I could find anything (`swagger-codegen` even removes fields of this type). Do you have any reference to how this works?